src/classiceditorui.js

```diff
@@ -88,6 +99,29 @@ export default class ClassicEditorUI {
 				}
 
 				return Promise.all( promises );
+			} )
+			.then( () => {
+				// Listen on the keystrokes from the main UI.
+				this.keystrokes.listenTo( this.view.element );
+
+				// Listen on the keystrokes from the floating panels, toolbars and the such.
+				this.keystrokes.listenTo( this.view._bodyCollectionContainer );
```

Comment:
> You can neither focus it if you have focus in a dialog or dropdown.

As for dialog, it makes sense because there's no way to reach the toolbar from the dialog in v4.

As for dropdown it also makes sense because... the dropdown is a part of the toolbar your about to focus.

But given the non–blocking approach that we use it the UI (floating balloons etc.), it's not a big deal if you access the toolbar using the keystroke from one or another. At least at this moment, I see no constraints.